src/fluree/db/connection.cljc

```diff
@@ -369,7 +369,7 @@
                               headers)
                res          (<? (xhttp/get url {:request-timeout 5000
                                                 :headers         headers*
-                                                :output-format   #?(:clj  :binary
+                                                :output-format   #?(:clj  :text
                                                                     :cljs :json)}))]
```

Comment:
Why `:text` for CLJ but `:json` for CLJS?

Reply:
When we're instantiating a db on the client, we make a call to `reify-db`, which fetches the db root via the default-read-fn, here. It then takes the result and passes the result to the serializer to deserialize into edn.

Our clj `xhttp/get` call with an `:output-format :json` automatically deserializes into edn, which our json serde can't process. By using `:output-format :text` we get the raw json string, which the json serde can happily deserialize into edn.

Reply:
Now that I think about it, `xhttp/get` is wrong to return edn when we ask for json. I think I'll fix it there instead of working around it.